techniques/failures/F96.html

```diff
@@ -2,148 +2,114 @@
-            <li><a href="F95">Previous Technique: F95</a></li>
-            <li><a href="F97">Next Technique: F97</a></li>
+            <li><a href="../#techniques" title="目次">目次</a></li>
+            <li><a href="../#introduction" title="イントロダクション">イントロダクション</a></li>
+            <li><a href="F95">前の達成方法: F95</a></li>
+            <li><a href="F97">次の達成方法: F97</a></li>
          </ul>
       </nav>
       <nav class="navtoc">
-         <p>On this page:</p>
+         <p>このページのコンテンツ</p>
          <ul id="navbar">
-            <li><a href="#important-information">Important Information about Techniques</a></li>
-            <li><a href="#applicability">Applicability</a></li>
-            <li><a href="#description">Description</a></li>
-            <li><a href="#examples">Examples</a></li>
-            <li><a href="#related">Related Techniques</a></li>
-            <li><a href="#tests">Tests</a></li>
+            <li><a href="#important-information">達成方法に関する重要な情報</a></li>
+            <li><a href="#applicability">適用（対象）</a></li>
+            <li><a href="#description">解説</a></li>
+            <li><a href="#examples">事例</a></li>
+            <li><a href="#related">関連する達成方法</a></li>
+            <li><a href="#tests">検証</a></li>
          </ul>
       </nav>
-      <h1>Failure due to the accessible name not containing the visible label text</h1>
+      <h1>F96: 達成基準 2.5.3 の失敗例 - "アクセシブルな名前 (accessible name)" が可視ラベルのテキストを含んでいない。</h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>
+            これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>All technologies that include interactive controls (such as links or form inputs).</p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/label-in-name">Success Criterion 2.5.3: Label in Name</a> (Failure)</span>.
-            </p>
+            <h2>適用（対象）</h2>
+            <p>インタラクティブなコントロール (リンクやフォーム入力など) を含むすべてのテクノロジー。</p>
```

Comment:
```suggestion
            <p>インタラクティブなコントロール (リンクやフォーム入力など) を含むすべての技術。</p>
```

ここで言うtechnologiesはWCAG用語集のそれになります https://waic.jp/docs/WCAG21/#dfn-technologies